src/Symfony/Component/Serializer/Normalizer/DateIntervalNormalizer.php

```diff
@@ -70,17 +71,16 @@ public function hasCacheableSupportsMethod(): bool
      *
      * @return \DateInterval
      *
-     * @throws InvalidArgumentException
-     * @throws UnexpectedValueException
+     * @throws NotNormalizableValueException
      */
     public function denormalize($data, string $type, string $format = null, array $context = [])
     {
         if (!\is_string($data)) {
-            throw new InvalidArgumentException(sprintf('Data expected to be a string, "%s" given.', get_debug_type($data)));
+            throw NotNormalizableValueException::createForUnexpectedDataType('Data expected to be a string.', $data, [Type::BUILTIN_TYPE_STRING], $context['deserialization_path'] ?? null, true);
```

Comment:
This should use `string` instead of `Type::BUILTIN_TYPE_STRING` for the expected type as the PropertyInfo component might not be installed (and yes, we have the same bug in other normalizers)